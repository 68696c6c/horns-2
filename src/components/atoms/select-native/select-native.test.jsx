/* global describe, it, expect */
import React from 'react'

import { render } from 'setup-test'

import Select from '.'

describe('Select', () => {
  it('should render as default', () => {
    const { container } = render(<Select id="example" name="example" />)
    expect(container).toMatchSnapshot()
  })
})
