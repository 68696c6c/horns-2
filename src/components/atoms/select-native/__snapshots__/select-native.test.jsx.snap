// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Select should render as default 1`] = `
.emotion-0 {
  border-top-width: 2px;
  border-top-style: solid;
  border-bottom-width: 2px;
  border-bottom-style: solid;
  border-left-width: 2px;
  border-left-style: solid;
  border-right-width: 2px;
  border-right-style: solid;
  background: rgb(255,255,255);
  color: rgb(1,1,1);
  border-color: rgb(216,216,216);
  cursor: text;
  border-top-left-radius: 4px;
  border-top-right-radius: 4px;
  border-bottom-left-radius: 4px;
  border-bottom-right-radius: 4px;
  font-family: Helvetica,sans-serif;
  font-size: medium;
  font-style: normal;
  font-weight: 400;
  line-height: 1em;
  padding-top: 8px;
  padding-bottom: 8px;
  padding-left: 8px;
  padding-right: 8px;
  display: -webkit-inline-box;
  display: -webkit-inline-flex;
  display: -ms-inline-flexbox;
  display: inline-flex;
  box-sizing: content-box;
  vertical-align: middle;
  height: 1em;
  min-width: 1em;
  -webkit-appearance: none;
  -moz-appearance: none;
  appearance: none;
  cursor: pointer;
}

.emotion-0:disabled,
.emotion-0.inactive {
  cursor: text;
  -webkit-text-decoration-line: none;
  text-decoration-line: none;
}

.emotion-0:not(:disabled):not(.inactive):hover {
  -webkit-text-decoration-line: none;
  text-decoration-line: none;
}

.emotion-0:not(:disabled):not(.inactive):active {
  -webkit-text-decoration-line: none;
  text-decoration-line: none;
}

.emotion-0::-ms-expand {
  display: none;
}

<div>
  <select
    class="control emotion-0"
    cursor="text"
    name="example"
    placeholder=""
  />
</div>
`;
